server/services/alexa/lib/alexa.onExecute.js

```diff
@@ -36,6 +50,35 @@ function onExecute(body) {
           f.type === DEVICE_FEATURE_TYPES.LIGHT.BINARY,
       );
       value = writeValues['Alexa.PowerController'](directiveName);
+      nameOfAlexaFeature = 'powerState';
+      break;
+    case 'Alexa.BrightnessController':
```

Comment:
I think you should introduce constants for switch cases, and load them in `SUPPORTED_DIRECTIVES_ON_EXECUTE` in `server/lib/gateway/gateway.handleAlexaMessage.js` file.
It can be "hard" to maintain both separetly.

Reply:
Fixed